### ISSUE TITLE
Default language not working, always English

### ISSUE THREAD

**Issue by JessinhoW** [stephanrauh/ngx-extended-pdf-viewer#878]
Hello,

I want nl-be as my default language. 
language is set to 'nl-BE' and useBrowserLocale is set to true.
![image](https://user-images.githubusercontent.com/25430059/129572562-33d98497-2596-4718-8449-279d7f15a101.png)

In the demo app at 
https://pdfviewer.net/extended-pdf-viewer/i18n
I looked and in my opninion the language should be es-ES. 

Am I doing something wrong? My language won't change and is English no matter what I do.
There are no errors in console/network calls.

Kind regards

**Comment by stephanrauh** [comment]
When I select Belgian in the demo, it doesn't look bad:

![image](https://user-images.githubusercontent.com/3045767/129586160-6a342c56-1b06-46ca-8ab3-c6cf0970acce.png)

Maybe the `locale` folder is missing in your `assets` folder?

**Comment by stephanrauh** [comment]
BTW: the screenshot is only 100 pixels high. That's a bug I've already fixed, but didn't copy the bugfix to the showcase yet.

**Comment by JessinhoW** [comment]
Hi, thank you for the fast reply, in the demo it is working indeed, after you select nl-BE but I want it just as default. I already checked that the locale folder is present. Tomorrow I will play around with settimeout to hide show the viewer but that seems not ideal to just set 1 default language. Is there maybe a demo/plunker where the default language is something else then English?

**Comment by stephanrauh** [comment]
When I set `[language]="'nl-BE'"`, the demo uses Belgian from start. However, since you only need a single language, maybe there's a better alternative. Have a look at https://github.com/stephanrauh/ngx-extended-pdf-viewer/tree/main/projects/ngx-extended-pdf-viewer#internationalization. Click the small triangle to expand the explanation on i18n and scroll down to "Inlining (aka embedding) the language files". You can add the translation file to the index.html (or any other HTML file that's part of the DOM when the PDF viewer is initializing). As a result, the PDF viewer needs two requests less to inititialize.

**Comment by stephanrauh** [comment]
I've added a demo showing the second approach to the showcase. To my surprise, the default locale setting of the JSON snippet is ignored (probably a bug). You need to set `[language]="'nl-BE'"` explicitly.

If you don't see the updated demo, clear your browser cache.

**Comment by JessinhoW** [comment]
For some reason it is not working for me. I forgot to mention I have a custom assests folder (but I don't think that's the problem) The script tag is present in the html, I have set useBrowserLocale to false but the language won't change :( . Some extra information I receive the src as a Blob from a rest service. But even if just set an empty viewer with only the language property and useBrowserLocale to false, the language won't change. (I tried different languages and also different browsers). I will just keep english for the moment.

**Comment by stephanrauh** [comment]
Does the network tab show any errors?

If all else fails: you can debug the code. It's a bit difficult because it's highly asychronous. The file you'll want to debug is l10n.js.

**Comment by JessinhoW** [comment]
Unfortunately no erros in teh network tab. Thank you for the debug instructions.

**Comment by stephanrauh** [comment]
At the moment, I've run out of ideas how to help you. It's always hard without seeing the actual source code. Maybe you can boil down your project to a reproducer? I. e. a simple project without third-party dependencies that shows the problem?

**Comment by stephanrauh** [comment]
Did you manage to get i18n up and running?

**Comment by JessinhoW** [comment]
I debugged for several hours/days but couldn't find out why it won't work. 
I debugged  the demo and my own application and in the demo after this line in I10n.js:
![image](https://user-images.githubusercontent.com/25430059/131791654-55d371eb-5c63-45e1-a5b6-b4afa4965ccf.png)

After that line I come in this function with as rawtext a lot of import statements
![image](https://user-images.githubusercontent.com/25430059/131793591-72583bc1-943a-4318-b66e-b3ff4edaa83d.png)

In my own application I don't come in teh function parse raw lines with all the import statements. I can't figure out why. Maybe it is the same problem because my pdfjs scripts are hosted on my own cdn. For the moment I kept the language on English.
